src/xunit.performance.api/Profilers/Etw/EtwProcess.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Xunit.Performance.Api.Profilers.Etw
+{
+    /// <summary>
+    /// Provides information about a profiled process.
+    /// </summary>
+    public sealed class EtwProcess
+    {
+        /// <summary>
+        /// Name of the process.
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Unique identifier associated with this process.
+        /// </summary>
+        public int Id { get; set; }
+
+        /// <summary>
+        /// The time that the associated process was started.
+        /// </summary>
+        /// <remarks>
+        /// It will be null if the process started before the ETW session was enabled.
+        /// </remarks>
+        public DateTime? Start { get; set; }
```

Comment:
Recommend being more descriptive with the names - StartDateTime or StartDate and ExitDateTime or ExitDate.